applications/ConvectionDiffusionApplication/python_scripts/python_solvers_wrapper_convection_diffusion.py

```diff
@@ -12,31 +12,38 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
         raise Exception("input is expected to be provided as a Kratos Parameters object")
 
     solver_type = solver_settings["solver_type"].GetString()
+    time_integration_method = "implicit / not defined"
+    if solver_settings.Has("time_integration_method"):
+        if solver_settings["time_integration_method"].GetString() == "explicit":
+            time_integration_method = solver_settings["time_integration_method"].GetString()
```

Comment:
In here:
1. Remove "implicit / not defined". It is either implicit or not defined, but no both at the same time.
2. If the `Has` is `true` save the provided `time_integration_method`. If not, set the `implicit` as default and
  2.1 Throw a warning specifying this
  2.2 And add it to the solver_settings dict (not only the auxiliary variable)

Reply:
Ok, thanks. I was unsure about the best strategy to follow.